### ISSUE TITLE
ML Notebook

### ISSUE THREAD

**Issue by louwjlabuschagne** [louwjlabuschagne/disaster-response-pipeline#2]
Finish the ML Notebook

**Comment by louwjlabuschagne** [comment]
# ML Notebook

For the machine learning portion, you will split the data into a training set and a test set. Then, you will create a machine learning pipeline that uses NLTK, as well as scikit-learn's Pipeline and GridSearchCV to output a final model that uses the message column to predict classifications for 36 categories (multi-output classification). Finally, you will export your model to a pickle file. After completing the notebook, you'll need to include your final machine learning code in train_classifier.py.
